integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/api/CallMediatorNoContentBackendAPI.xml

```diff
@@ -0,0 +1,8 @@
+<api xmlns="http://ws.apache.org/ns/synapse" name="CallMediatorNoContentBackendAPI" context="/CallMediatorNoContentBackendAPI">
```

Comment:
Add license headers?

Reply:
Do we really need licencing for configs?  I checked few other configs and most of them do not have license headers.

Reply:
Yes, all files which can be commented should have the license header.

Reply:
Fixed. Thanks.